libs/SalesforceFileLogger/SalesforceFileLogger/Classes/Logger/SFSDKLogger.m

```diff
@@ -213,7 +213,7 @@ - (void)d:(Class)cls message:(NSString *)message {
 
 - (void)logInternal:(Class)cls level:(DDLogLevel)level message:(NSString *)message  {
     NSString *tag = [NSString stringWithFormat:kLogIdentifierFormat, self.componentName, cls];
-    DDLogMessage *logMessage = [[DDLogMessage alloc] initWithMessage:message level:level flag:DDLogFlagForLogLevel(level) context:0 file:self.componentName function:nil line:0 tag:tag options:0 timestamp:[NSDate date]];
```

Comment:
this method is deprecated so I switched it over to the same init the deprecated one was calling https://github.com/CocoaLumberjack/CocoaLumberjack/blob/af4973721172dd7850a42a58a9ffcec0ec82e5a9/Sources/CocoaLumberjack/DDLog.m#L1028